dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -258,3 +258,105 @@ CWWKD1023.extra.param.explanation=Query conditions that define \
+ entity property does not represent text.
+
+CWWKD1027.anno.missing.prop.name=CWWKD1027E: Parameter {0} of the {1} method of \
+ the {2} repository interface is annotated with an {3} annotation that does not \
+ specify an entity property name as its value. The annotation must either specify \
+ the entity property name as its value or the -parameters compile option must be \
+ enabled and the parameter must have the same name as the entity property.
+CWWKD1027.anno.missing.prop.name.explanation=Parameters of repository update \
+ methods must correspond to entity properties.
+CWWKD1027.anno.missing.prop.name.useraction=Update the repository method to \
+ indicate the entity property to which the parameter pertains.
+
+CWWKD1028.first.exceeds.max=CWWKD1028E: The {0} method of the {1} repository \
+ interface requests the first {2} results, which exceeds the maximum allowed \
+ number of results: {3}.
+CWWKD1028.first.exceeds.max.explanation=The maximum number of query results is \
+ limited by system constraints and the value specified.
+CWWKD1028.first.exceeds.max.useraction=Reduce the requested maximum number of \
+ query results to a smaller value.
+
+CWWKD1029.first.neg.or.zero=CWWKD1029E: The {0} method of the {1} repository \
+ interface requests the first {2} results. The requested maximum number of \
+ results to return must be positive.
+CWWKD1029.first.neg.or.zero.explanation=The maximum number of query results \
+ must be a positive number.
+CWWKD1029.first.neg.or.zero.useraction=Update the repository method to specify \
+ a positive maximum number of results.
+
+CWWKD1030.ql.lacks.entity=CWWKD1030E: The entity name is missing from the {0} \
+ query that is specified for the {1} method of the {2} repository. In Jakarta \
+ Data Query Language, {3} queries should be formed like: {4}.
+CWWKD1030.ql.lacks.entity.explanation=The query contains a clause that requires \
+ the entity name.
+CWWKD1030.ql.lacks.entity.useraction=Update the query for repository method to \
+ include the entity name.
+
+CWWKD1031.ql.similar.entity=CWWKD1031E: The {0} method of the {1} repository \
+ interface specifies a query that requires a {2} entity name that is not found \
+ but is a close match for the {3} entity name. The query is: {4}.
+CWWKD1031.ql.similar.entity.explanation=The query might have a typo in its \
+ reference to the entity name.
+CWWKD1031.ql.similar.entity.useraction=Review the query to ensure the correct \
+ entity name is used.
+
+CWWKD1032.ql.unknown.entity=CWWKD1032E: The {0} method of the {1} repository \
+ interface specifies a query that requires a {2} entity name that is not found. \
+ Check if {2} is the name of a valid entity. To enable the entity to be found, \
+ give the repository a lifecycle method that is annotated with one of the {3} \
+ annotations and supply the entity as its parameter. Alternatively, have the \
+ repository extend DataRepository or another built-in repository interface, \
```

Comment:
change
extend DataRepository
to
extend the DataRepository interface

Reply:
Updating as recommended.